packages/block-editor/src/components/inspector-controls-sub-sheet/README.md

```diff
@@ -0,0 +1,94 @@
+# InspectorControlsSubSheet
+
+Inspector Controls Sub Sheet allows for adding controls inside the React Native bottom sheet settings. 
+
+### Usage
+
+```jsx
+/**
+ * External dependencies
+ */
+import { SafeAreaView, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { InspectorControlsSubSheet } from '@wordpress/block-editor';
+import { useNavigation } from '@react-navigation/native';
+import { useState } from '@wordpress/element';
+import { Icon, chevronRight } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import Cell from '../bottom-sheet/cell';
+import NavigationHeader from '../bottom-sheet/navigation-header';
+
+const ExampleControl = () => {
+	const [ showSubSheet, setShowSubSheet ] = useState( false );
+	const navigation = useNavigation();
+
+	const goBack = () => {
+		setShowSubSheet( false );
+		navigation.goBack();
+	};
+
+	const openSubSheet = () => {
+		navigation.navigate( InspectorControlsSubSheet.screenName );
+		setShowSubSheet( true );
+	};
+
+	return (
+		<InspectorControlsSubSheet
+			button={
+				<Cell
+					label={ 'Howdy' }
+					separatorType="none"
+					onPress={ openSubSheet }
+				>
+					<Icon icon={ chevronRight }></Icon>
+				</Cell>
+			}
+			showSheet={ showSubSheet }
+		>
+			<SafeAreaView>
+				<NavigationHeader
+					screen={ 'Howdy' }
+					leftButtonOnPress={ goBack }
+				/>
+				<View paddingHorizontal={ 16 }>
+					<Text>{ 'World' }</Text>
+				</View>
+			</SafeAreaView>
+		</InspectorControlsSubSheet>
+	);
+};
+
+export default ExampleControl;
+```
+
+### Props
+
+#### showSheet
+
+Controls the Sub Sheet content visibility.
+
+-   Type: `Boolean`
+-   Required: Yes
+
+#### navigationButton
```

Comment:
You may already be aware of this and have plans for further changes, but I wanted to point out my suggestions renamed this prop from `button` to `navigationButton`. So, we should likely either rename the prop in the component itself or revert the change to the documentation. Thanks!

Reply:
Fixed.